tagmanager/src/tm_source_file.c

```diff
@@ -109,6 +109,35 @@ gchar *tm_get_real_path(const gchar *file_name)
 	return NULL;
 }
 
+/* add argument list of __init__() Python methods to the class tag */
+static void update_python_arglist(const TMTag *tag)
```

Comment:
uctags doesn't do (and in fact cannot do) anything - it was added by Enrico in 1932d3fae938a21c1a0b22c4ae205f63f8ff8d0c. The purpose of it is for:

```
class Foo:
    def __init__(self, bar, baz):
        pass
```

to use the tag for its __init__() and copy its argument list to the tag of Foo so when you type

```
a = Foo(
```

you get completion based on the Foo's __init__(). Thing like that cannot be done in uctags because at the time __init__() is parsed, the Foo tag is already written to disk and cannot be modified to include the arguments of __init__().